pkgs/by-name/fi/firefly-iii-data-importer/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, unzip }:
+let
+  version = "1.5.2";
+in
+stdenv.mkDerivation rec {
+  pname = "firefly-iii-data-importer";
+  inherit version;
+
+  src = builtins.fetchurl {
+    url = "https://github.com/firefly-iii/data-importer/releases/download/v${version}/DataImporter-v${version}.zip";
+    sha256 = "sha256:1ssxhgd6x4lp8ak5zkgj02hvpvdsgxafli9syb9lwrkkr9z8lyyg";
+  };
```

Comment:
Can we build this from source instead of pulling in the built tarball?

Reply:
I'm having trouble building it from source, apparently the target class `view` isn't found, and I don't know if I missed any step in the build process as there are absolutely no documentation in the source code (or doc website).
I may dig deeper on it later, for now I cannot really do better